lib/jekyll-feed.rb

```diff
@@ -12,6 +12,14 @@ def config
       @context.registers[:site].config
     end
 
+    def path
+      if config["feed"] && config["feed"]["path"]
```

Comment:
So the way it is now:

```rb
if config["feed"] && config["feed"]["path"]
```

...won't blow up unless feed.path is set and incorrect (e.g. a hash instead of a string). I'm personally fine with this, as it's well documented how it should properly be configured.